translations/ja_JP.ts

```diff
@@ -44,7 +44,7 @@
     <message>
         <source>T</source>
         <translatorcomment>表示スペースの都合上、省略文字を利用</translatorcomment>
-        <translation>翻訳</translation>
```

Comment:
I forgot to explain detail, this patch fix the string cropped
the text on red line will be replaced to green line text.